CHANGELOG.rst

```diff
@@ -5,6 +5,12 @@ If a ``rowexpander`` does not return a row in the form of a ``dict``,
 ``Dimension.ensure`` now explicitly raises a ``TypeError`` with the name of the
 function set as the ``rowexpander``.
 
+``ymdhmsparser`` can now handle datetime.datetime as input. Any other input is
```

Comment:
Line breaks seems to be missing from **Unreleased** in version [rendered](https://github.com/chrthomsen/pygrametl/blob/e57b934f7f83528250b4666ea095582903495337/CHANGELOG.rst) by GitHub.

Reply:
Good catch. Fixed

Reply:
You're right. Fixed.

Reply:
Is str purposely not `str`? It looks like we have written string in CHANGELOG.rst when a we talk about a string and not the specific Python type.

Reply:
I suggest we also write "string. (GitHub issue #40)" for `ymdparser` to keep `ymdhmsparser` and `ymdparser` consistent.

Reply:
Sorry, thought I had done that. Fixed now.